doc/docs/FAQ.md

```diff
@@ -236,7 +250,11 @@ No. Currently, Meep only supports anisotropic, real-symmetric, permittivity tens
 
 ### When outputting the permittivity function to a file, I don't see any dispersive materials
 
-Only the real, frequency-independent part of ε/μ is written to an HDF5 file. As an example, many of the dispersive materials in the [materials library](Materials.md#materials-library) which have a broadband, complex, refractive index will appear as ε=1 in the output file. Thus, in order to verify the material geometry during debugging using visualization tools, etc., you may have to artificially adjust the `epsilon` value.
+Only the real, frequency-independent (i.e. non dispersive) part of ε/μ is written to an HDF5 file. As an example, many of the dispersive materials in the [materials library](Materials.md#materials-library) which have a broadband, complex, refractive index will appear as ε=1 in the output file. Thus, in order to verify the material geometry during debugging using visualization tools, etc., you may have to artificially adjust the `epsilon` value. If you want the frequency-dependent ε you must compute it yourself using the formula for [Lorentizian susceptbility](Materials.md#material-dispersion).
+
+### How do I model graphene or other 2d materials with single-atom thickness?
+
+For modeling 2d materials, you will need to use a one-pixel-thick [conductor](Materials.md#conductivity-and-complex). It is not necessary for the `resolution` to specify a pixel size of one atom because of [subpixel averaging](#can-subpixel-averaging-be-applied-to-dispersive-materials).
```

Comment:
You’ll also want to scale the conductivity with the resolution

Reply:
Fixed.